R/run.R

```diff
@@ -202,7 +202,6 @@ CmdStanRun <- R6::R6Class(
           if (self$args$save_latent_dynamics)
             self$latent_dynamics_files(include_failed = TRUE)
         )
-        message("Cleaning up CmdStanR temporary files")
```

Comment:
unrelated to the issue in question, but i don't think we actually need this message. it will just confuse people.

Reply:
Agree.